__tests__/server.test.js

```diff
@@ -0,0 +1,15 @@
+const request = require('supertest');
+
+const server = require('../server/index.js');
+
+describe('get /data', function() {
+  it('responds with data', function(done) {
+    request(server)
+      .get('/data')
+      .expect('Content-Type', /json/)
+      .expect((res)=>{
+        res.body.length > 2
+      })
+      .expect(200, done);
+  });
+});
```

Comment:
This code is invoking expect functions without assertions.

I suggest to you look the signature of **expect** function:
https://jestjs.io/docs/en/expect.html#expectvalue

Reply:
This is super test syntax I copied it from their  documentation. https://www.npmjs.com/package/supertest

Reply:
> This is super test syntax I copied it from their documentation. https://www.npmjs.com/package/supertest

Nice! I didn't know that! Thanks for telling me.